apps/web/playwright/booking/all-locations-regular-booking.e2e.ts

```diff
@@ -0,0 +1,120 @@
+
+      const scheduleBookingPage = await bookingPage.previewEventType();
+      await bookingPage.selectFirstAvailableTime(scheduleBookingPage);
+
+      location === "In Person (Attendee Address)" &&
+        (await bookingPage.fillBookingFormLocation(scheduleBookingPage, location));
+      location === "Attendee Phone Number" &&
+        (await bookingPage.fillBookingFormLocation(scheduleBookingPage, "+55 32 988776655", location));
+
+      await bookingPage.confirmBooking(scheduleBookingPage);
+      await bookingPage.clickReschedule(scheduleBookingPage);
+      await bookingPage.selectFirstAvailableTime(scheduleBookingPage);
+      await bookingPage.fillRescheduleReasonAndConfirm(scheduleBookingPage);
+      await bookingPage.assertBookingScheduled(scheduleBookingPage);
+      await bookingPage.cancelBookingWithReason(scheduleBookingPage);
+    });
+  }
+
+  for (const customLocation of customLocationTypes) {
+    test(`Add the ${customLocation} location and schedule a meeting (display on booking page checked)`, async ({
+      page,
+      bookingPage,
+    }) => {
+      await bookingPage.selectLocation(customLocation);
+      await bookingPage.verifyTextVisibility(customLocation);
+      await bookingPage.fillLocationInformation(customLocation, page);
+      await bookingPage.updateEventType();
+
+      const scheduleBookingPage = await bookingPage.previewEventType();
+
+      await bookingPage.assertLocationAdded(scheduleBookingPage, customLocation);
+      await bookingPage.selectFirstAvailableTime(scheduleBookingPage);
+
+      await bookingPage.assertLocationAdded(scheduleBookingPage, customLocation);
+      await bookingPage.confirmBooking(scheduleBookingPage);
+      await bookingPage.clickReschedule(scheduleBookingPage);
+      await bookingPage.selectFirstAvailableTime(scheduleBookingPage);
+      await bookingPage.fillRescheduleReasonAndConfirm(scheduleBookingPage);
+      await bookingPage.assertBookingScheduled(scheduleBookingPage);
+      await bookingPage.cancelBookingWithReason(scheduleBookingPage);
+    });
+  }
+
+  for (const customLocation of customLocationTypes) {
+    test(`Add the ${customLocation} location and schedule a meeting (display on booking page not checked)`, async ({
+      page,
+      bookingPage,
+    }) => {
+      await bookingPage.selectLocation(customLocation);
+      await bookingPage.verifyTextVisibility(customLocation);
+      await bookingPage.fillLocationInformation(customLocation, page);
+      await bookingPage.updateEventType();
+
+      const scheduleBookingPage = await bookingPage.previewEventType();
+
+      await bookingPage.assertLocationAdded(scheduleBookingPage, customLocation);
+      await bookingPage.selectFirstAvailableTime(scheduleBookingPage);
+
+      await bookingPage.assertLocationAdded(scheduleBookingPage, customLocation);
+      await bookingPage.confirmBooking(scheduleBookingPage);
+      await bookingPage.clickReschedule(scheduleBookingPage);
```

Comment:
We don't need to test the entire flow of rescheduling and cancelling when testing locations since we test this elsewhere.